netqasm/lang/instr/trapped_ion.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+from typing import List, Union
+
+import numpy as np
+
+from netqasm.lang.operand import Immediate, Operand, Register, Template
+
+from ...util.quantum_gates import get_rotation_matrix
```

Comment:
I prefer using absolute imports; relative imports sometimes give problems depending on the location you call Python scripts from